cppwinrt/component_writers.h

```diff
@@ -988,9 +988,22 @@ namespace winrt::@::implementation
     static void write_generated_static_assert(writer& w)
     {
         auto format = R"(
-// Note: Remove this static_assert after copying these generated source files to your project.
-// This assertion exists to avoid compiling these generated source files directly.
-static_assert(false, "Do not compile generated C++/WinRT source files directly");
+// WARNING: This file is automatically generated from IDL. Do not directly
+// add this file to your project, as any changes you make will be lost.
+//
+// If this is a new file:
```

Comment:
"If this is a new file" is unclear about what "this" is. Literally, "this" is the auto-generated file, which is naturally new. But the "this" in this sentence really means "the corresponding file in your project".

```suggestion
// To add a copy of this file to your project:
```